src/python/py/models/builder.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
             },
             "dml": {},
             "webgpu": {},
+            "openvino": {},
```

Comment:
Should be "OpenVINO" (and the other two should be changed, but that's not your fault)